README.md

```diff
@@ -1,26 +1,35 @@
 # Node VCX Agency
-- Implementation of [LibVCX](https://github.com/hyperledger/indy-sdk/tree/master/vcx) (V3 protocol) compatible Agency (more specifically 
-[Mediator Agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md)
-in Aries terminology).
+- Implementation of [AriesVCX](https://github.com/hyperledger/aries-vcx/) compatible 
+[Mediator Agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md).
 
 - Mediator Agency can service many users, providing them endpoint where 3rd parties can deliver encrypted messages. 
 Assuming the sending party has properly end2end encrypted the message for recipient, according to Aries connection  
 protocol, the agency itself can't decrypt content of received messages.
 
+- VCX Agency implements 2 notifications mechanism. 
+  1. Webhooks - The agency call specified url when a particular agent receives a message. This is useful when using
```

Comment:
"The agency calls a specified url when an agent receives a message. This is useful when using AriesVcx on a server."